internal/dao/rs.go

```diff
@@ -127,3 +127,50 @@ func (r *ReplicaSet) Rollback(fqn string) error {
 
 	return nil
 }
+
+// GetOwners returns the owners of the resource.
+func (r *ReplicaSet) GetOwners(path string) ([]OwnerInfo, error) {
```

Comment:
Perhaps we should use Controllers here to keep with k8s terminology?

Reply:
This might be a bit misleading. `ownerReferences` specifically has a bool attribute `controller`. We might navigate to an owner which is not a controller. So owner would be a more suitable term as it is more generic.

Reply:
As there are a couple of comments regarding this naming, I would wait for your feedback before tackling the others.

Reply:
Lets keep owners. That makes perfect sense!